Storage/SQL/SQLiteHistory.swift

```diff
@@ -174,7 +174,7 @@ fileprivate struct FrecentHistoryImpl : FrecentHistory {
         "LEFT JOIN \(ViewHistoryVisits) h ON b.url = h.url"
 
         db.withConnection { connection in
-            let existsQuery = "SELECT COUNT(*) FROM sqlite_master WHERE type = 'table' and name = '\(AwesomebarBookmarksTempTable)'"
+            let existsQuery = "SELECT COUNT(*) FROM sqlite_temp_master WHERE type = 'table' and name = '\(AwesomebarBookmarksTempTable)'"
```

Comment:
changing `sqlite_master` to `sqlite_temp_master` should have been in the previous commit. 
That is the place to look for temp tables